script/install.sh

```diff
@@ -390,14 +411,20 @@ Run \`$ROOT/install.sh --help\` for usage information.
 resolve_latest() {
     case $COMPILER in
         dmd)
-            local url=http://downloads.dlang.org/releases/LATEST
-            logV "Determing latest dmd version ($url)."
-            COMPILER="dmd-$(fetch $url)"
+            local mirrors=(
+                "http://downloads.dlang.org/releases/LATEST"
+                "http://ftp.digitalmars.com/LATEST"
+            )
+            logV "Determing latest dmd version (${mirrors[0]})."
+            COMPILER="dmd-$(fetch "${mirrors[*]}")"
             ;;
         dmd-beta)
-            local url=http://downloads.dlang.org/pre-releases/LATEST
-            logV "Determing latest dmd-beta version ($url)."
-            COMPILER="dmd-$(fetch $url)"
+            local mirrors=(
+                "http://downloads.dlang.org/pre-releases/LATEST"
+                "http://ftp.digitalmars.com/LATEST_BETA"
+            )
+            logV "Determing latest dmd-beta version (${mirrors[0]})."
+            COMPILER="dmd-$(fetch "${mirrors[*]}")"
```

Comment:
Think this should be `[@]` expanded, so fetch really gets 2 arguments.
```bash
cnt() { echo $#; }
args=('entry with' 'spaces')
cnt "${args[*]}" # 1
cnt "${args[@]}" # 2
cnt ${args[*]} # 3
cnt ${args[@]} # 3
```
As with "$@" you always want to pass arguments in an array using `"${ary[@]}".